plot_continuous_monitor.py

```diff
@@ -226,9 +153,171 @@ def get_util_for_disk(disk_utils, disk):
       return disk_util[disk]
   return None
 
+def gnuplot_data_line(data_hash):
+    """ Helper function to order continuous montior data for gnuplot to use """
+    return [data_hash['time'],
```

Comment:
This looks very possibly error prone in the future.  Instead, what about changing line 111 to construct a list instead of a dictionary (so just change the curly braces to square ones). Then, you can use dict() to make it into a dictionary to pass into the matplot lib stuff, and you can do something like [a[0] for a in list] to get the data for gnuplot.